pootle/apps/pootle_store/unit/timeline.py

```diff
@@ -236,7 +238,7 @@ def grouped_entries(self):
         grouped_entries.reverse()
         return grouped_entries
 
-    @property
+    @cached_property
     def submissions(self):
```

Comment:
i think keep this as a @property that returns a qs, and add a cached_property `submission_values` or similar that turns it into a regular list